universal-application-tool-0.0.1/app/controllers/admin/ProgramAdminManagementController.java

```diff
@@ -86,8 +87,24 @@ public Result update(Http.Request request, long programId) {
           roleService.makeProgramAdmins(
               programId, ImmutableSet.copyOf(manageAdminForm.getAdminEmails()));
       Result result = redirect(routes.AdminProgramController.index());
-      if (maybeError.isPresent()) {
-        return result.flashing("error", maybeError.get().message());
+
+      if (maybeError.isPresent()){   
+        Optional<Program> program =
+            programRepository.lookupProgram(programId).toCompletableFuture().join();
+        if (program.isEmpty()) {
+          return notFound(String.format("Program with ID %s was not found", programId));
+        } else {
+          ImmutableList<String> programAdmins =
+              programRepository.getProgramAdministrators(programId).stream()
+                  .map(Account::getEmailAddress)
+                  .collect(toImmutableList());
+          return ok(
```

Comment:
nit: empty line before `return` statement and rest of method body

Reply:
Done.